sockeye/arguments.py

```diff
@@ -160,6 +160,10 @@ def add_model_parameters(params):
                               type=int_greater_or_equal(1),
                               default=1,
                               help='Minimum frequency of words to be included in vocabularies. Default: %(default)s.')
+    model_params.add_argument('--joint-vocab',
```

Comment:
So the only reason for a joint vocab would be if we want to use a shared embedding, right? How about having a parameter that determines if we tie source and target embeddings and if that parameters is True we internally build the shared vocab. What do you think?